dali/pipeline/data/types_test.cc

```diff
@@ -33,20 +33,24 @@ namespace {
   static constexpr size_t DUMMY_ARRAY_SIZE = 42;
 }
 
-#define TYPENAME_FUNC(type)                     \
+#define TYPENAME_FUNC(type, typename)           \
```

Comment:
Don't keywords as macro arguments - even when they happen to _not_ collide.

Reply:
Done